src/hotspot/cpu/riscv/macroAssembler_riscv.hpp

```diff
@@ -735,12 +767,12 @@ class MacroAssembler: public Assembler {
   typedef void (MacroAssembler::* jal_jalr_insn)(Register Rt, address dest);
   typedef void (MacroAssembler::* load_insn_by_temp)(Register Rt, address dest, Register temp);
 
-  void wrap_label(Register r, Label &L, Register t, load_insn_by_temp insn);
```

Comment:
Better to remove the `load_insn_by_temp` declaration from file macroAssembler_riscv.hpp as it is not used anywhere after this change.

`typedef void (MacroAssembler::* load_insn_by_temp)(Register Rt, address dest, Register temp);`

Reply:
Fixed